### PR TITLE
my-sites: tidyng my-sites/navigation

### PR DESCRIPTION
**Pull request by retrofox**
* minor jsdoc improvements
* es6ify

### CODE REVIEW

**Comment by gwwar** [comment]
:shipit:

File: client/my-sites/navigation/navigation.jsx [hunk 1]
```diff
@@ -23,27 +22,20 @@ module.exports = React.createClass( {
 	 * Calculate the height of the sites list based on how many sites
 	 * the user has to be displayed
 	 *
-	 * @return {number}
+	 * @return {Number} height of the sites list
 	 */
-	getSitesHeight: function() {
+	getSitesHeight() {
```
Comment (mtias) [inline client/my-sites/navigation/navigation.jsx]: We can remove this whole function.

File: client/my-sites/navigation/navigation.jsx [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
-	SitePicker = require( 'my-sites/picker' ),
-	Sidebar = require( 'my-sites/sidebar' );
+import observe from 'lib/mixins/data-observe';
+import SitePicker from 'my-sites/picker';
+import Sidebar from 'my-sites/sidebar';
 
-var SITE_HEIGHT = 66,
-	EXTRA_SITE_ITEMS_HEIGHT = 200,
-	MAX_SIDEBAR_HEIGHT = 1200;
+const SITE_HEIGHT = 66,
```
Comment (mtias) [inline client/my-sites/navigation/navigation.jsx]: And these constants.

File: client/my-sites/navigation/navigation.jsx (hunk 1 above)
Comment (retrofox) [inline client/my-sites/navigation/navigation.jsx]: :+1:

**Pull request merged**